docs/docs/statistics/power.mdx

```diff
@@ -116,108 +124,61 @@ Frequently asked questions:
 3. What if my experiment is low-powered? Should I still run it? The biggest cost to running a low-powered experiment is that your results will probably be noisy, and you will face ambiguity in your rollout/rollback decision. That being said, you will probably still have learnings from your experiment.
 4. What does "N/A" mean for my MDE result? It means there is no solution for the MDE, given the current number of weekly users, control mean, and control variance. Increase your number of weekly users or reduce your number of treatment variations.
 
-## GrowthBook implementation
+## GrowthBook implementation (frequentist)
+
+Below we describe high-level technical details of our implementation. All technical details can be found [here](/statistics/powerTechnical).
```

Comment:
Thanks for the updates here. I still think we need a bit more signposting and clear language.

Can we say something like

```
## GrowthBook implementation

For both Bayesian and frequentist engines, we produce two key statistics:

1. Power - After running the experiment for a given number of weeks and for a hypothesized effect size, what is the probability of a statistically significant result?
2. Minimum Detectable Effect (MDE) - After running an experiment for a given number of weeks, what is the smallest effect that we can detect as statistically significant 80% of the time.

Each engine arrives at these values differently...tech details link... etc...

### Frequentist engine

...clear signposting for **Power** and **MDE**...

### Bayesian engine

...clear signposting for **Power** and **MDE**...

Reply:
thanks, I agree it's helpful for the key equations to really pop.  Changes implemented.